### PR TITLE
Added angularUtils package with autoupdate

### PR DESCRIPTION
**Pull request by AndreiTelteu**
I have added the angularUtils package.
This is useful for me because it does a great job integrating disqus in my app.
I did `npm test` and there was no error.

### CODE REVIEW

**Comment by AndreiTelteu** [comment]
The test says `./ajax/libs/angularUtils/0.0.1/src/directives/disqus/dirDisqus.js does not exist but is referenced in package.json!`
Do i have to add all the files manually ?

**Comment by AndreiTelteu** [comment]
I deleted previous commit, and i have added the files for the current version, but i don't understand something: the CircleCI test says `/ajax/libs/angularUtils/0.0.1/src/directives/disqus/dirDisqus.js does not exist but is referenced in package.json`
The readme file says that `basePath will be ignored when copying over to the CDN`, but it is not ignored on the test. What I did wrong ?

File: ajax/libs/angularUtils/package.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
+{
+	"name": "angularUtils",
+	"filename": "src/directives/disqus/dirDisqus.js",
```
Comment (PeterDaveHello) [inline ajax/libs/angularUtils/package.json]: This is the reason why build failed.

**Comment by AndreiTelteu** [comment]
Updated and working ! thanks @PeterDaveHello

**Comment by PeterDaveHello** [comment]
I think I can not confirm its version number, would you please confirm it with the author?

**Comment by AndreiTelteu** [comment]
I'll send him a message.

**Comment by PeterDaveHello** [comment]
BTW, we don't hosting html files.

**Comment by drewfreyling** [comment]
Closing due to no response.

**Pull request closed without merging**